src/osx/core/bitmap.cpp

```diff
@@ -134,6 +134,66 @@ private :
     wxDC*         m_selectedInto;
 };
 
+typedef wxObjectDataPtr<wxBitmapRepresentationData> wxBitmapRepresentationDataPtr;
+WX_DEFINE_TYPEARRAY( wxObjectDataPtr<wxBitmapRepresentationData>, wxBitmapRepresentationArray, dummy );
+
+class WXDLLEXPORT wxBitmapRefData : public wxGDIRefData
+{
+public:
+    wxBitmapRefData()
+    {
+        
+    }
+    
+    wxBitmapRefData(const wxBitmapRefData &tocopy) :
+        m_representations( tocopy.m_representations)
+    {
+        
+    }
+
+    wxBitmapRefData(wxBitmapRepresentationData *rep)
+    {
+        AddRepresentation(rep);
+    }
+    
+    void AddRepresentation(wxBitmapRepresentationData *rep)
+    {
+        m_representations.push_back(wxBitmapRepresentationDataPtr(rep));
+    }
+
+    void AddRepresentations(const wxBitmap& other)
+    {
+        const wxBitmapRefData* otherData = other.GetBitmapData() ;
+        for ( size_t i = 0 ; i < otherData->m_representations.size() ; ++i )
+        {
+            m_representations.push_back(otherData->m_representations.at(i));
+        }
+    }
+
+    wxBitmapRepresentationData* GetDefaultRepresentation()
+    {
+        return m_representations.at(0).get();
```

Comment:
Throwing from here when there are no representations doesn't seem like a good idea (if it's intentional).

We should probably just return `NULL` if the vector is empty (and if it isn't, it's safe to use `[0]` instead of `.at(0)`)